plugins/ps/functions

```diff
@@ -9,8 +9,21 @@ print_dokku_scale_file() {
   done < "$DOKKU_SCALE_FILE"
 }
 
+get_procfile() {
+  local APP="$1"; local IMAGE_TAG="$3"; local IMAGE=$(get_app_image_name $APP $IMAGE_TAG); local DOKKU_PROCFILE="$DOKKU_ROOT/$APP/DOKKU_PROCFILE"
```

Comment:
Also, why is `IMAGE_TAG` set to `$3` and not `$2`? We never call `get_procfile()` with more than 1 arg. Additionally that brings up the question about supporting image tags.